rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -72,6 +73,22 @@ def decode_liquidation(self, context: DecoderContext) -> None:
         token and part of the collateral deposited is lost too. Those two events happen as
         transfers in a transaction started by the liquidator."""
 
+    def _is_aave_token_contract(self, queried_address: 'ChecksumEvmAddress | None') -> bool:
+        """Utility function for checking if a queried_address is an aave token.
+        Returns True if it is an aave token, or False if it is not."""
+        if queried_address is None:
+            return False
+
+        with GlobalDBHandler().conn.read_ctx() as cursor:
+            return (protocol := cursor.execute(
+                'SELECT protocol FROM evm_tokens WHERE identifier=?',
+                (evm_address_to_identifier(
+                    address=queried_address,
+                    chain_id=self.evm_inquirer.chain_id,
+                    token_type=EvmTokenKind.ERC20,
+                ),),
+            ).fetchone()) is not None and protocol[0] == self.counterparty
```

Comment:
tgis should be as staticmethod in the globaldb that takes the 3 values as agument and returns you the counterparty so you can later compare